main_test.go

```diff
@@ -195,6 +195,17 @@ func TestPublishersEndpoints(t *testing.T) {
 			},
 			expectedContentType: "application/json",
 		},
+		{
+			query: "POST /v1/publishers - INVALID PAYLOAD",
+			body:  `{"URL":"https://www.example.com", "email":"error"}`,
+			headers: map[string][]string{
+				"Authorization": {goodToken},
+				"Content-Type":  {"application/json"},
+			},
+			expectedCode:        422,
+			expectedBody:        `{"title":"can't create Publisher","detail":"invalid json","status":422,"validationErrors":[{"field":"email","rule":"email","value":"error"}]}`,
```

Comment:
This should say `invalid format` to be consistent with the other resources, I'll add a remark at https://github.com/italia/developers-italia-api/issues/12